utils/Status.js

```diff
@@ -40,6 +40,7 @@ const STATUSMESSAGE = {
   FORBIDDEN: "Forbidden !",
   NOT_FOUND: "Not found !",
   EVENT_SLUG_ALREADY_EXISTS: "Event slug already exists",
+  EVENT_NOT_FOUND: "Event not found !",
```

Comment:
Please revert this change, we have the `NOT_FOUND` tag already. Please use that.

Reply:
Have this extra `EVENT_NOT_FOUND` being removed ?
We don't want to remove `EVENT_SLUG_ALREADY_EXISTS` by the way.